README.md

```diff
@@ -243,6 +247,24 @@ https://github.com/brendon/positioning/blob/main/lib/positioning/advisory_lock.r
 
 If you have any concerns or improvements please file a GitHub issue.
 
+### Opting out of Advisory Lock
+
+You can opt out of Advisory Lock by setting `use_advisory_lock` as `false` when declaring positioning:
+
+```ruby
+belongs_to :list
+positioned on: :list, use_advisory_lock: false
+```
+
+When disabling Advisory Lock, you still want to avoid race conditions. There are multiple ways to achieve that, one of them is locking the parent record manually:
+
+```ruby
+list = List.create(name: "My list")
+list.with_lock do
+  list.items.create(name: "My list item")
```

Comment:
It might be helpful to also show an `update` and `destroy` case here if they're particularly different scenarios?